dannielRolfe/file-1.js

```diff
@@ -0,0 +1,255 @@
+//ecxercise #1
+
+//a)
+"1" == 1 /*Output: Always True The operand "==" represents Equal To. Because, javascript will try to help and convert the string "1" to 1 the output will be true.*/
+
+//b)
+"1" === 1 /*Output: Always False The operand "===" represents Equal Value and Equal Type. Unlike, "1"==1, where there is a type conversion, this operand is checking for equality. This operand is more precise than the equal to operand and therefor the better choice when checking for a more precise equality.*/
+
+//c) --- Incorrect 
+x == 'x' /*Output:true when x equals a number or a string Javascript will convert 'x' to a number; however, if x equals an object or undefined then the output is false*/
+
+//correct answer - false UNLESS x is literally string 'x'
+
+//d)
+x == (x+'')
+
+number == (number + '') /*Output:True  When x equals a number the first x converts to a string. */
+
+null == (null+'') /*Output:False Null is an object and javascript does not know how to add an object to a string. */
+
+undefined == (undefined + '') /*Output:false Undefined is "Undefined" and javascript does not know hot interpret undefined + string.*/
+
+"string" == ("string" + '') /*Output:true a string is equal to a string*/
```

Comment:
Some edge cases: If `x = NaN`, this will be false, as `NaN` is not equal to anything, including itself; and if `x` is a boolean, it will autoconvert to s string when added to a string.